src/buildercore/terraform.py

```diff
@@ -69,6 +68,8 @@
   "cache_status":"%{regsub(fastly_info.state, "^(HIT-(SYNTH)|(HITPASS|HIT|MISS|PASS|ERROR|PIPE)).*", "\\\\2\\\\3") }V"
 }"""
 
+FASTLY_LOG_FORMAT_ESCAPED = FASTLY_LOG_FORMAT.replace('%', '%%')
```

Comment:
I think this is a terraform 0.11 vs 0.13 thing, rather than logging format version 1 vs 2 thing but I'm not sure as I get errors on both v1 and v2 format versions.

https://docs.fastly.com/en/guides/custom-log-formats